.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
```

Comment:
I was hoping to get the tests passing in this pull request as well unless it is a huge diff.